lisa/sut_orchestrator/libvirt/schema.py

```diff
@@ -15,6 +15,12 @@ class CloudInitSchema:
     # Additional values to apply to the cloud-init user-data file.
     extra_user_data: Optional[Union[str, List[str]]] = None
 
+# Configuration options for Ignition provisioning for the VM.
+@dataclass_json()
+@dataclass
+class IgnitionSchema:
+    # Additional values to apply to the Ignition user-data file.
+    extra_user_data: Optional[Union[str, List[str]]] = None
```

Comment:
In cloud-init, the file is literally named `user-data`. Are the equivalent files also called `user-data` for Ignition?